geom/opticks_raindrop.gdml

```diff
@@ -80,8 +93,8 @@
       <fraction n="0.996003" ref="Ar400x560299384330"/>
     </element>
     <material name="G4_AIR0x560299383bc0" state="gas">
-      <property name="RINDEX" ref="RINDEX0x560299349730"/>
-      <property name="GROUPVEL" ref="GROUPVEL0x560299349110"/>
+      <property name="RINDEX" ref="RINDEX0x5602993a1df1"/>
+      <property name="GROUPVEL" ref="GROUPVEL0x56029938bb90"/>
```

Comment:
What was the reason for changing this properties for G4_AIR? Was it needed to see hits?

Reply:
So, just to confirm this change is not relevant for getting G4 hits, correct?

Reply:
Correct, for G4Hits irrelevant. But we want Optickshits too.